src/pmdas/perfevent/configparser.h

```diff
@@ -36,6 +36,7 @@ typedef struct pmcsetting {
     int cpuConfig;
     double scale;    /* Currently, only used by derived events */
     int need_perf_scale;  /* Currently, only used by derived events */
+    unsigned long rawcode;  /* Currently, only used by raw events */
```

Comment:
This is going to patch-conflict with the recent change from @anjutsudhakar I think - could you pull those changes in (merging upstream now) and update this PR please?

Reply:
Sure.